uint64 utime
uint8 in_fault
RomanFaultState fault_status
RomanWaypoint state

#TODO: RobotiqState hand1_state
#TODO: RobotiqState hand2_state
#TODO: TalonState base_state
